data/modules/CrewContracts/CrewContracts.lua

```diff
@@ -208,9 +208,17 @@ local onChat = function (form,ref,option)
 		form:ClearFace()
 		form:Clear()
 		form:SetTitle(l.CREW_FOR_HIRE)
-		form:SetMessage("\n"..l.POTENTIAL_CREW_MEMBERS:interp({station=station.label}))
+		local numCrewInThisStation = 0
 		for k,c in ipairs(crewInThisStation) do
-			form:AddOption(l.CREWMEMBER_WAGE_PER_WEEK:interp({potentialCrewMember = c.name,wage = Format.Money(c.estimatedWage)}),k)
+			numCrewInThisStation = numCrewInThisStation + 1
+		end
```

Comment:
Of what I can see, the `crewInThisStation` is an array, since it's only using `insert`, so `#crewInThisStation` should give you the array lenght. Just for future reference.